Runtime/Graphics/CGraphics.cpp

```diff
@@ -34,6 +34,9 @@ SViewport g_Viewport = {
     0, 0, 640, 480, 640 / 2.f, 480 / 2.f, 0.0f,
 };
 u32 CGraphics::g_FrameCounter = 0;
+u32 CGraphics::g_Framerate = 0;
+u32 CGraphics::g_FramesPast = 0;
+std::chrono::time_point<std::chrono::high_resolution_clock> CGraphics::g_FrameStartTime = std::chrono::high_resolution_clock::now();
```

Comment:
I might do
 ```c++
using frame_clock = std::chrono::high_resolution_clock;
```
and then you can use 
```c++
frame_clock::now()
frame_clock::time_point
``` 
etc

Reply:
I had forgotten about `using`, definitely much cleaner now!